client/src/components/core/ms-toggle/MsGridListToggle.vue

```diff
@@ -24,9 +21,6 @@
       @click="$emit('update:modelValue', modelValue === DisplayState.Grid ? DisplayState.List : DisplayState.Grid)"
     >
       <ion-icon :icon="list" />
-      <span v-if="modelValue === DisplayState.List">
-        {{ $t('WorkspacesPage.viewDisplay.list') }}
```

Comment:
Is this text used somewhere else? If not, it should be removed from the translation files.